#include <catch2/catch_test_macros.hpp>

TEST_CASE("dummy tests", "[dummy]")
{
    SUCCEED();
}
